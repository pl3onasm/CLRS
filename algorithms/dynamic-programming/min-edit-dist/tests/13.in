identical
identical

1 2 3 4 5 6