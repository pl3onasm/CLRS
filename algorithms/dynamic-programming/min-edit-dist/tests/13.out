Minimum edit distance: 9

Optimal operation sequence:

Source:
  identical
Copy i:
  i | dentical
Copy d:
  id | entical
Copy e:
  ide | ntical
Copy n:
  iden | tical
Copy t:
  ident | ical
Copy i:
  identi | cal
Copy c:
  identic | al
Copy a:
  identica | l
Copy l:
  identical | 
Target:
  identical
